etl/steps/data/garden/ihme_gbd/2023-10-04/leading_causes_child_deaths.meta.yml

```diff
@@ -0,0 +1,61 @@
+tables:
```

Comment:
Before defining tables, you could add dataset metadata:
```
dataset:
  title: ...
  update_period_days: 365
```

Reply:
Ah I thought this would be propagated from snapshot?

Reply:
The title may be, but `update_period_days` is a field that needs to be defined in `garden`.